### DISCUSSION TITLE
Producing a Video with Markers

### DISCUSSION THREAD

**Discussion by RomilaImran** [talmolab/sleap#1839] [Help!] [answered]
Hello, this may sound like a silly question, but how does one produce videos with markers in MP4? I have trained, ran inference, and now I want to have the original video but with the markers. There is an export labels option, but I believe it allows me to export frames only. Is there an option to automatically create a video on GUI?

Thank you so much

**Comment by talmo** [comment, marked as answer]
Hi @RomilaImran,

Yep, from the GUI you can go to **View**  → **Render Video Clip with Instances...** to render a quick visualization. If you hold **Shift** while dragging on the seekbar, you can also select a subclip within your video.

Let us know if you have any other questions!

Cheers,

Talmo
